connector/dialect.tdd

```diff
@@ -145,6 +145,16 @@
       <unagg-formula>(CASE WHEN %1 IS NULL THEN NULL ELSE 0 END)</unagg-formula>
       <argument type='int' />
     </function>
+    <function group='aggregate' name='VAR' return-type='real'>
+      <formula>(STDDEV_SAMP(%1) * STDDEV_SAMP(%1))</formula>
+      <unagg-formula>(CASE WHEN %1 IS NULL THEN NULL ELSE 0.0 END)</unagg-formula>
```

Comment:
This unagg-formula should be NULL as the sample of a single value returns null: 
https://www.mongodb.com/docs/manual/reference/operator/aggregation/stdDevSamp/#single-value

```suggestion
      <unagg-formula>NULL</unagg-formula>
```

Reply:
Yes, you are right.
I originally had NULL, but then I found this in [Tableau docs](https://tableau.github.io/connector-plugin-sdk/docs/dialect)   : 
> unagg-formula element The unagg-formula (unaggregated formula) element is an optional child of function elements. It should be specified only if the function is part of the aggregate group. Unaggregated formulas should represent a reasonable way of expressing the aggregate of a single value. For example, for average, it should be the value itself. **For variance, it should be 0.**
Looks like NULL makes more sense.
Changed it to NULL. Thanks for pointing this out!